daemon/commit.go

```diff
@@ -70,6 +74,21 @@ func (daemon *Daemon) Commit(container *Container, repository, tag, comment, aut
 	}
 	defer rwTar.Close()
 
+	blobStore := daemon.graph.BlobStore()
+	diffWriter, err := blobStore.NewWriter(crypto.SHA256)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get new writer for rootfs diff: %s", err)
+	}
+	defer func() {
+		if err != nil {
```

Comment:
Right. If this defer was in some nested scope I would have used a different name for the functions returned error, maybe `fErr`.